readme.md

```diff
@@ -24,7 +25,10 @@ with FullscreenWindow() as window:
             elif c == '<SPACE>':
                 a = FSArray(window.height, window.width)
             else:
-                s = repr(c).decode()
+			    if sys.version_info[0] == 2:
```

Comment:
The indention seems to be off.

Reply:
Weird, it looks fine in my original edit, but it could be something to do with the markdown-mode switching Emacs (org-mode tends to be a little better for that sort of thing). I'll have to have a closer look a bit later.